metabrainz/admin/quickbooks/quickbooks.py

```diff
@@ -0,0 +1,34 @@
+from intuitlib.client import AuthClient
+from quickbooks import QuickBooks
+from flask import request, current_app
+
+def init(app):
+    '''
+    Create global auth client that manages QuickBooks sessions.
+    '''
+
+    if not app.config["QUICKBOOKS_CLIENT_ID"]:
+        app.logger.warn("No QUICKBOOKS_CLIENT_ID specified, not setting up QB invoice feature.")
```

Comment:
What happens to the admin views when this is not specified? we still register the views, do they ISE? Maybe we shouldn't register them?